_posts/2020-12-04-a-new-way-to-learn-and-discover-quarkus.adoc

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: A new way to learn & discover Quarkus (even for the Quarkus veterans)
+date: 2020-12-04
+tags: extensions codestarts quickstart
+synopsis: All our tooling has been updated and can now generate Quarkus application with example code showing the true power of the selected extensions...
+author: adamevin
+---
+
+What’s nice about this new feature is that it’s useful for fresh Quarkus developers as well as experienced ones. This is because of the wide range (and growing) of Quarkus extensions, you simply can’t know it all.
+
+Would you blame me if I say that we developers are a tiny bit lazy? We also like well documented frameworks.. But reading docs is a bit boring.. Right?
```

Comment:
I don't grok what this paragraph is trying to say ? is it necessary ? that even though we are lazy we manage to write docs for quarkus ? :)

Reply:
OK I'll rephrase it, I am trying to say, that's we went a step further than the doc and made the code directly available in the IDE (no doc needed :)..

Reply:
okey, writing that out explicitly would be better - no reason to let the reader be guessing :)